src/client/components/rich-text/rich-text.stories.js

```diff
@@ -3,12 +3,29 @@ import { withKnobs, text } from '@storybook/addon-knobs'
 import { withReadme } from 'storybook-readme'
 import readme from './readme.md'
 import RichText from './'
+import ContentSection from '../content-section'
+import alwaysSecureJson from '../../static/data/en/pages/always-secure.json'
```

Comment:
We try to keep the stories in the guide independent of the CMS. So I'd rather have some static content here. If you have a lot of content you can add a `.stories.json` next to the `.stories.js` file, like other components already have. This story should also work without the `ContentSection`, so please leave it out. That component already has its own stories.

(and you left a `console.log`)